modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/registry/ObjectServiceUpgradeStepRegistrator.java

```diff
@@ -479,6 +479,11 @@ protected String[][] getTableAndPrimaryKeyColumnNames() {
 
 		registry.register(
 			"10.0.0", "10.0.1", new ObjectDefinitionPortletIdUpgradeProcess());
+
+		registry.register(
+			"10.0.1", "10.1.0",
+			UpgradeProcessFactory.alterColumnType(
+				"ObjectEntry", "externalReferenceCode", "VARCHAR(255)"));
```

Comment:
This is required because we are setting FDS name as ERC. This is the key part of wiring. The problem is, FDS names can grow large, because they include portlet namespace (see any `*FDSNames.java` file).

Reply:
That's ok to me, @markocikos ! Thanks for the contribution.